README.md

```diff
@@ -204,3 +204,54 @@ Requires `GITLAB_ACCESS_TOKEN` variable to be set, which is an access token with
   - `SSH_PRIVATE_KEY` A private key of a user which can push to Pantheon
   - `SSH_KNOWN_HOSTS` The result of running `ssh-keyscan -H codeserver.dev.$PANTHEON_SITE_ID.drush.in`
   - `TERMINUS_PLUGINS` Comma-separated list of Terminus plugins to be available (optional)
+
+## Tugboat Integration
+
+Add the following to `composer.json` to add Tugboat configuration:
+
+```json
+{
+    "extra": {
+        "drainpipe": {
+            "tugboat": {}
+        }
+    }
+}
+```
+
+The PHP version is autodetected based on the composer requirements.
+
+A `provider` object allows the upstream host to be determined and to determine
+if Tugboat downsyncs from the provider or does a fresh Drupal site install.
+
+### Acquia
+```json
+{
+    "extra": {
+        "drainpipe": {
+            "tugboat": {
+                "provider": {
+                    "host": "acquia",
+                    "downsync": true
```

Comment:
It turns out we don't actually have a task command to downsync from Acquia. @marcoscano or @hawkeyetwolf think that's something you'd like to contribute based on your other project work?